instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -85,13 +85,10 @@ def test_colors_and_images_applied(self):
                 'SIMPLETHEME_SASS_OVERRIDES': [
                     {'variable': 'link-color',
                      'value': '#003344', },
-                    # TODO: These are specific to Ginkgo and can be removed
```

Comment:
@giovannicimolin, the enclosed styles can be removed when the corresponding ones above are removed.